guardian/newsletters-nx
# Various Updates

## What does this change?

- Update Draft Created email text not to say `launched`
- Add Nginx config
- Update Setup scrip
- Update Dependencies
- Address Linting issues with returned promises (Post deps update)

## How to test

- Test script adds nginx config as expected and that app is served at https://newsletters-tool.local.dev-gutools.co.uk/
- Check email content on creating a  draft
- Test that App stands update and it's possible to create & edit newsletters

## How can we measure success?

Everything should still work

## Have we considered potential risks?

No major version updates so should be OK
